lib/reservation.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'date_range'#take note this needs to be included at the top
+# require 'Date'
+
+module Hotel
+
+  class Reservation
+
+    attr_reader :check_in, :check_out, :date_range, :room_num, :cost, :reservation_array
+
+    def initialize(check_in, check_out, room_num, cost = 200)
+      @reservation_array = []
```

Comment:
I'm not sure I understand what the `reservation_array` is for. Looks like you just put all the attributes for the reservation in it? I don't see that it's used by any of the rest of your code.

Reply:
The more I think about this the more it concerns me. Next time we've got project time, come see me - I want to hear what you're thinking.